internal/convert.go

```diff
@@ -31,7 +31,9 @@ type Conv struct {
 	SrcSchema                map[string]schema.Table             // Maps source-DB table name to schema information.
 	Issues                   map[string]map[string][]SchemaIssue // Maps source-DB table/col to list of schema conversion issues.
 	ToSpanner                map[string]NameAndCols              // Maps from source-DB table name to Spanner name and column mapping.
+	ToSpannerFkIdx           map[string]FkeyAndIdxs              // Maps from source-DB table name to Spanner table name, foreign keys and index mappings.
```

Comment:
consider adding `json:"-"` struct tags so that we dont export these to the session file.

Reply:
_ToSpanner_ and _ToSource_ are being used in the code base functionally, putting them into a different _"audit_" object will lead to a number of cascading changes throughout the codebase. Adding only _ToSpannerFkIdx_, _ToSourceFkIdx_ to _audit_ will seperate the central storage of column name mapping from the foreign key and index mappings. This will make the mappings slightly more difficult to track.

To avoid peppering them through the conv, it is perhaps better to put them along with the other variables that don't affect the functionality in the _conv_ struct only. eg -along with  _SchemaConversionDuration_, _DataConversionDuration_.